jekyll/_layouts/classic-docs.html

```diff
@@ -1,58 +1,138 @@
+							<div id="instant-hits"></div>
+						</div>
+						<div role="tabpanel" class="tab-pane" id="search-results-orbs">
+							<div id="instant-hits-orbs"></div>
+						</div>
+						<div role="tabpanel" class="tab-pane" id="search-results-cimgs">
+							<div id="instant-hits-cimgs"></div>
+						</div>
+					</div>
+		</div>
+	</div>
+	<div class="main-body">
+		{% include sidebar.html %}
+    <div class="sidebar-offset">
+      <div class="article-container docs" id="main">
+        {% if page.collection == "ccie" %}{% include v2-banner.html %}{% endif %}
+        {% if page.collection == "cci1" %}{% include v2-dev-banner.html %}{% endif %}
+        {% if page.page-type == "index" %}<h2>{{ page.title }}</h2>{% endif %}
+
+
+        <article>
+          <!-- Title and version badges: -->
+          <div>
+            {% if page.page-type != "index" and page.page-type != "homepage" %}<h1 id="{{ page.title | downcase | split: " " | join: "-" }}">{{ page.title }}</h1>{% endif %}
           </div>
-          {% for version in page.version %}
-          <span class="badge badge-primary">
-            <button class="server-version-badge" aria-describedby="tooltip">{{ version }}</button>
-            <!-- <img src="{{ site.baseurl }}/assets/img/docs/{{ version }}-badge.png" height="24">  -->
-          </span>
-          {% endfor %}
-        </div>
-        {% endif %}
-        {% if page.untranslated %}
-          {% include_cached translation_in_progress.html site=site page=page %}
-        {% endif %}
-        {{ content }}
-        {% if page.collection != 'cci1' and page.page-type != "index" and page.layout == "classic-docs" or page.layout == "enterprise" %}
-          {% include_cached doc-footer.html page=page site=site %}
+
+            {% if page.version %}
+            <div class="server-version-wrapper">
+              <div id="tooltip" role="tooltip" class="server-version-popover">
+                This document is applicable to the following:
+                <div id="arrow" data-popper-arrow></div>
+              </div>
+              {% for version in page.version %}
+              <span class="badge badge-primary">
+                <button class="server-version-badge" aria-describedby="tooltip">{{ version }}</button>
+                <!-- <img src="{{ site.baseurl }}/assets/img/docs/{{ version }}-badge.png" height="24">  -->
+              </span>
+              {% endfor %}
+            </div>
+            {% endif %}
+
+
+          {% if page.untranslated %}
+            {% include translation_in_progress.html %}
+          {% endif %}
+
+          {{ content }}
+
+          {% if page.collection != 'cci1' and page.page-type != "index" and page.layout == "classic-docs" or page.layout == "enterprise" %}
+            {% include doc-footer.html %}
+          {% endif %}
+
+        </article>
+
+      </div><!-- .article-container -->
+
+        {% if page.side_toc != "none" %}
```

Comment:
Would it work to just use jekyll-toc's `toc` param for this conditional, instead of a separate one?